src/main/resources/CodyBundle.properties

```diff
@@ -79,6 +79,12 @@ context-panel.tree.root=repositories-root
 context-panel.tree.node-chat-context=Chat Context
 context-panel.tree.node-local-project=Local Project
 context-panel.tree.node-remote-repos=Remote Repo(s)
+context-panel.add-repo-dialog.title=Add Remote Repo
+context-panel.add-repo-dialog.error-empty-url=Remote repository URL cannot be empty
+context-panel.add-repo-dialog.error-invalid-url=Remote repository URL must be valid
+context-panel.add-repo-dialog.error-no-repo=Remote repository not found on the server
+context-panel.add-repo-dialog.url-input-label=<html><b>Repo URL</b><html>
+context-panel.add-repo-dialog.url-input-help=<html><span>Example: <b>github.com/sourcegraph/cody</b><br>Only repos indexed on customer instance are valid.<br><br>Contact yours Sourcegraph admin to add missing repo.<span><html>
```

Comment:
NIT/maybe in topic: I think it's still not clear what 'contact sg admins' means. I imagine myself as a new user and search in Google 'sourcegraph how to index repo,' but it's hard to find an answer. AFAIK, we need to write on Discord that we want indexing. Maybe we should add a link here?

Reply:
FYI it's enterprise only feature.